samples/boards/nxp/mimxrt1170_evk_cm7/magic_addr/sample.yaml

```diff
@@ -9,5 +9,6 @@ common:
     - pytest
 tests:
```

Comment:
As the board specific overlay was renamed to flexram.overlay, should it be added to the sample.yaml as an extra overlay?

Reply:
@butok , sorry, my mistake in fixing a build error for simulator, let me update